appveyor.yml

```diff
@@ -0,0 +1,72 @@
+# This file was automatically generated by conda-smithy. To update a component of this
```

Comment:
The normal way to do add this is to [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ). Did that not work for some reason?